mkosi/__init__.py

```diff
@@ -7533,8 +7538,8 @@ def find_ovmf_vars() -> Path:
 
     die("Couldn't find OVMF UEFI variables file.")
 
-
-def run_qemu(args: MkosiArgs) -> None:
+@contextlib.contextmanager
+def run_qemu_cmdline(args: MkosiArgs) -> Iterator[Sequence[str]]:
```

Comment:
Same here
```suggestion
def run_qemu_cmdline(args: MkosiArgs) -> Iterator[List[str]]:
```
this has the added benefit (if I might call it that), that later on at line 101 in `machine.py`, we get complaints, because both strings and lists are sequences, so mypy doesn't notice the type of the variable changing from a list to a string.